### ISSUE TITLE
training accuracy decrease slightly at the start of every epoch 

### ISSUE THREAD

**Issue by chenjx1005** [apache/incubator-mxnet#3126]
I train a Inception-v2 network on my dataset with around 600 classes and 1M images. 
At the beginning  every thing is fine. The training accuracy increases consistently. 
However after some epochs, the training accuracy begins to drop slightly at the start of every epoch.
![image](https://cloud.githubusercontent.com/assets/3807357/17924569/c304fa52-6a1b-11e6-852a-1e8dea5c48ed.png)

**Comment by chenjx1005** [comment]
the accuracy vs iteration curve of the training. we can find a  clear drop at the start of every epoch.
![loss_curve](https://cloud.githubusercontent.com/assets/3807357/17954232/0e4dcdfa-6aac-11e6-8e64-bfa92f23fb66.jpg)

**Comment by pluskid** [comment]
Not sure why this happens. But if you could provide some more details like that optimizers, learning rate, etc. are you using, maybe somebody could try to have some guessing of potential reasons.

**Comment by phunterlau** [comment]
This issue is closed due to lack of activity in the last 90 days. Feel free to reopen if this is still an active issue. Thanks!
